paper.tex

```diff
@@ -78,9 +78,9 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{INTRODUCTION}
 
-In the advent of connected vehicles and autonomous transportation, equipping consumer vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems can be prohibitively expensive.
-Technologies such as Radar, LIDAR and high-end cameras that are typically used for collision prediction are likely too expensive for most consumers.
-On the other hand, dash-cams, which are cheap video cameras that can be mounted to a car's dashboard, have become increasingly affordable and prevalent as a means to capture a driver’s perspective.
+In the advent of connected vehicles and autonomous transportation, we are equipping vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems.
+Using these vehicles to simulate collision situations has a two-fold cost. Both the car and expensive / experimental sensors are put at risk.
+However, we can collect collision data naturally in real driving scenarios through the use of dash-cams, cheap video cameras that can be mounted to a car's dashboard.
```

Comment:
I would change the last comma to a hyphen:

"However, we can collect collision data naturally in real driving scenarios through the use of dash-cams - cheap video cameras that can be mounted to a car's dashboard."